modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console to see how the traffic moves around the application, in the menu bar, click the *Application launcher* > *Kiali*.
+
+Select the project for your service in the *Namespace* field. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console. If you have installed the Bookinfo example, select `bookinfo`.
```

Comment:
FYI - If you have deployed Kiali on OpenShift you can use the 'openshift' login option (this is the default option if you’re using OpenShift). With this option, users log in to Kiali with the OpenShift OAuth login. What users can access in Kiali will now be based on their user roles in OpenShift using the Kubernetes RBAC.     (So, I would say "If necessary, log in to the Kiali console with the same user name and password....."  And rather than take them right to the Graph page, you might say something like, "When you first log in, you see the Overview page which displays all the namespaces in your mesh that you have permission to view."